config/200-webhook-clusterrole.yaml

```diff
@@ -41,22 +41,23 @@ rules:
       - "apps"
     resources:
       - "deployments"
-    verbs:
-      - "get"
+      - "deployments/finalizers"
+    verbs: &everything
+      - get
+      - list
+      - watch
+      - create
+      - update
```

Comment:
Does the webhook need full permissions on deployments and finalizers? Maybe sufficient to add a new `apiGroups` stanza, leaving the previous `deployments` one in place:

```yaml
- apiGroups:
  - "apps"
  resources:
    - "deployments/finalizers"
  verbs:
    - update
```